pkg/status/health/global.go

```diff
@@ -15,18 +15,20 @@ var readinessOnlyCatalog = newCatalog()
 var startupOnlyCatalog = newStartupCatalog()
 
 // RegisterReadiness registers a component for readiness check with the default 30 seconds timeout, returns a token
-func RegisterReadiness(name string) *Handle {
-	return readinessOnlyCatalog.register(name)
+func RegisterReadiness(name string, once bool) *Handle {
+	return readinessOnlyCatalog.register(name, once)
 }
 
 // RegisterLiveness registers a component for liveness check with the default 30 seconds timeout, returns a token
-func RegisterLiveness(name string) *Handle {
-	return readinessAndLivenessCatalog.register(name)
+func RegisterLiveness(name string, once bool) *Handle {
+	return readinessAndLivenessCatalog.register(name, once)
 }
 
 // RegisterStartup registers a component for startup check, returns a token
 func RegisterStartup(name string) *Handle {
-	return startupOnlyCatalog.register(name)
+	// we register the startup probe with once=false because once a startup is healthy
```

Comment:
```suggestion
	// Startup health checks are registered with once=false because, by design, they are only checked once.
```

Reply:
I think this can be a bit misleading

`they are only checked once` might be confusing, the reader might think it means that it is checked only once, and if it fails, we don't check it anymore and it dies. 

I updated it to something that is more obvious.

Let me know what you think

Reply:
Checked together, resolved 👍